quinn-proto/src/connection/mod.rs

```diff
@@ -35,6 +35,8 @@ use crate::{
 
 mod assembler;
 
+mod pacing;
```

Comment:
Looks like this module isn't actually included in this commit?

Reply:
Sorry about that!